### DISCUSSION TITLE
faucet calibration

### DISCUSSION THREAD

**Discussion by cmartins88** [filecoin-project/community#189] [Q&A]
Hi everybody,

how long does it take to receive the funds?

![image](https://user-images.githubusercontent.com/676892/122271941-e62e0c00-ced7-11eb-950e-1ea62d0cfdb9.png)
(bafy2bzacedooragvmg7vaiql4q2xadi63rohbiyf2jxqphperksabexmh67tm)

I'm a newbie, and I just want to configure a server to test filecoin mining with lotus, and I think something is wrong because I'm waiting hours for miner creation and wallet funds by faucet...

![image](https://user-images.githubusercontent.com/676892/122273111-fc889780-ced8-11eb-8977-12a04376e05a.png)
(worker wallet: t3w4weys7xvnvxo6pxt7yedgribjijawodwp2euw27t3zjsg5sudh7axgvrzvsdjvdojemxuouv5drrci3cdgq)

Lotus is working and syncing...
![image](https://user-images.githubusercontent.com/676892/122272955-d4009d80-ced8-11eb-9ceb-bd8f245021ae.png)

Thanks.

**Comment by longfeiWan9** [comment]
> how long does it take to receive the funds?

Normally, filecoin produce a tipset (set of blocks in a certain block heigh) every 30 seconds. If everything is normal, you will receive your funds in 30 seconds or a bit longer. But if the network is congested with messages, it might take longer.

BTW, the calibration testnet was upgraded lately. If you are still using older version of lotus (e.g 1.9.0), you might need to upgrade your lotus node to 1.10.0 to avoid any problem.

**Comment by jennijuju** [comment]
It depends really. You can use `lotus state search-msg` or  `lotus state wait-msg` to check if the message has landed on chain. 
Btw- and Longfei mentioned, calibration net has been upgraded to network v13 https://github.com/filecoin-project/community/discussions/74#discussioncomment-848818 with actor v5. I would suggest you to check your lotus version to make sure you are on lotus v1.10.0-rc1 and higher or check `lotus state get-actor <your miner is>`  and confirm  `Code` is v5.
If not, likely you are on the wrong chain.
To fix that, you need to run `lotus chain sethead —epoch=321500` with your daemon running, then shut down your miner and node, check out the latest release, rebuild lotus and restart you daemon.

join #fil-net-calibration-discuss on filecoin slack if you have further questions!

**Reply by cmartins88** [reply]
Thanks you so much, you're both right @wanglongfei88 and @jennijuju. I'm a newbie, like a said, and I follow the rules by the book and on network status (https://network.filecoin.io/#calibration) the release tag that I should used is v1.8.0 ... and it is not! I checkout the latest one, make everything again, and that's it!

Before I wrote on this forum, I tried to access to slack with google authentication unsuccessfully ... I guess I don't have permissions. Is it necessary to ask someone for that?

I really want to participate in this community...since the end of last year I've been following the project, and it's fabulous!

Thanks

**Reply by jennijuju** [reply]
Good catch- I’ll update the page later today.
As for slack, it’s open for everyone https://github.com/filecoin-project/community#chat . Try the link here and lmk if it works
